mlrun/feature_store/api.py

```diff
@@ -94,12 +94,13 @@ def get_offline_features(
     target: DataTargetBase = None,
     run_config: RunConfig = None,
     drop_columns: List[str] = None,
-    start_time: Union[str, pd.Timestamp] = None,
-    end_time: Union[str, pd.Timestamp] = None,
+    start_time: Union[str, datetime] = None,
+    end_time: Union[str, datetime] = None,
     with_indexes: bool = False,
     update_stats: bool = False,
     engine: str = None,
     engine_args: dict = None,
+    spark_service: Optional[str] = None,
```

Comment:
Usually we don't want to add a new parameter in the middle (or rather, not the end) of the param list, since it might break existing code which called this function without named args. I understand the desire to keep it with the other params that are engine-related, and also there's a very slim chance that someone actually called this method with all its params, but suggest you still put that in the end of the list.